src/ucs/debug/debug.c

```diff
@@ -1108,7 +1109,7 @@ static inline void ucs_debug_save_original_sighandler(int signum,
     kh_value(&ucs_signal_orig_action_map, hash_it) = oact_copy;
 
 out:
-    pthread_spin_unlock(&ucs_kh_lock);
+    ucs_spin_unlock(&ucs_kh_lock);
```

Comment:
`ucs_spinlock_release`

Reply:
I can't find `ucs_spinlock_release ` in the latest master (c303724fd7bc85ef9609f92c1b5cd575b6a1e1f4)
What am I missing?